src/main/java/gg/solarmc/kitpvp/messaging/parsers/KillstreakParser.java

```diff
@@ -0,0 +1,17 @@
+package gg.solarmc.kitpvp.messaging.parsers;
+
+import gg.solarmc.kitpvp.messaging.Parser;
+
+public class KillstreakParser implements Parser {
+
+    public KillstreakParser(String streakAmount) {
+        this.streakAmount = streakAmount;
+    }
+
+    private final String streakAmount;
```

Comment:
Fields should really be above constructors

Reply:
autogeneration is one hell of a drug